app/models/header_lookup.rb

```diff
@@ -1,15 +1,12 @@
 class HeaderLookup
-  LCSH_BASE_URI = "http://id.loc.gov/authorities/subjects/suggest/?q="
-  LCSH_ID_URI = "http://id.loc.gov/authorities/subjects/"
-  MESH_ID_URI = "https://id.nlm.nih.gov/mesh/"
-
   MEMOIZED_MESH_FILE = "memoized_mesh.txt"
   MEMOIZED_LCSH_FILE = "memoized_lcsh.txt"
   MEMOIZED_LCNAF_FILE = "memoized_lcnaf.txt"
 
-  SEARCHABLE_LCNAF_FILE = "subjects_lcnaf.jsonl"
-  SEARCHABLE_MESH_FILE = 'subjects_mesh.jsonl'
-  SEARCHABLE_LCSH_FILE = 'subjects_lcsh.jsonl'
+  SEARCHABLE_MESH_FILE = "subjects_mesh.jsonl"
+  SEARCHABLE_LCSH_FILE = "subjects_lcsh.jsonl"
+  # Needs to be in config because file is too large for VCS
+  SEARCHABLE_LCNAF_FILE = ENV["SUBJECTS_LCNAF"]
```

Comment:
LCNAF is just too big to include with the repo :( Frustrating, but it will work.

Reply:
So an LCNAF file will be copied onto the server?